web/satellite/src/components/account/ProfileArea.vue

```diff
@@ -60,6 +60,7 @@
                 height="56px"
                 :on-press="toggleDeleteAccountPopup"
                 is-deletion="true"
+                :is-disabled="true"
```

Comment:
This cannot fully stop user from performing delete action

Reply:
done